lynx/src/main/java/com/github/pedrovgs/lynx/model/Lynx.java

```diff
@@ -130,6 +139,15 @@ public synchronized void unregisterListener(Listener lynxPresenter) {
     listeners.remove(lynxPresenter);
   }
 
+  private void setFilters() {
+    lowerCaseFilter = lynxConfig.getFilter().toLowerCase();
+    try {
+      regexpFilter = Pattern.compile(lowerCaseFilter);
+    } catch (PatternSyntaxException exception) {
+      regexpFilter = null;
```

Comment:
We should log a message here using the Android logger. Just to let the user know and the developer there is something wrong with the regex in case we need it. A simple ``Log.d("LYNX", "...")`` should be enough.